ejb3/src/main/java/org/jboss/as/ejb3/timerservice/persistence/database/DatabaseTimerPersistence.java

```diff
@@ -337,6 +337,8 @@ private String identifyDialect(String name) {
                 unified = MSSQL;
             } else if (name.contains(SYBASE) || name.contains(JCONNECT)) {
                 unified = SYBASE;
+            } else {
+                EjbLogger.ROOT_LOGGER.unknownDatabaseName(name);
```

Comment:
`EJB3_TIMER_LOGGER` should be used here.

Reply:
Thanks, @chengfang did the changes.